README.md

```diff
@@ -27,54 +46,55 @@
     "time_intervals": <number>,
     "max_seating": <number>,
     "max_party_size": <number>
+    "max_time_range": <number>
   },
   "bookings": [
     {
-      "_id": <string>,
       "booking_time": <datetime>,
       "party_qty": <number>,
```

Comment:
general comment - use camelcase?

Reply:
I prefer to use camelCase on the front end and snake_case on the back end to help me keep track of where they live. I would prefer to continue using this convention since it makes it easier on me when having to context switch between front and back ends.